admin-data-explorer-web/src/main/scala/net/wiringbits/webapp/utils/ui/web/models/ColumnType.scala

```diff
@@ -8,11 +8,14 @@ object ColumnType {
   case object Date extends ColumnType
   case object Text extends ColumnType
   case object Email extends ColumnType
+  case object Image extends ColumnType
   case class Reference(referencedTable: String, source: String) extends ColumnType
 
   def fromTableField(column: TableColumn): ColumnType = {
     val isEmail = column.name.contains("email")
     val isDate = column.`type`.equals("timestamptz")
+    val isImage = column.`type`.equals("bytea")
```

Comment:
I just got to notice that all these matches should be case-insensitive (`equalsIgnoreCase`)